nubis/bin/consul2unixusers.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+
+import os
+import sys
+import pwd
+import ConfigParser
+import optparse
+import consul
+
+USERADD="/usr/sbin/useradd"
+CHOWN="/bin/chown"
+CHMOD="/bin/chmod"
+MKDIR="/bin/mkdir"
+DRYRUN=False
+
+_config = {}
+
+def load_config(config_file):
+    global _config
+    if not os.path.exists(config_file):
+        print "ERROR: Config file %s not found." % config_file
+        sys.exit(-1)
+    config = ConfigParser.RawConfigParser()
+    config.read(config_file)
+    if 'consul' not in config.sections():
+        print "ERROR: [consul] section not found in %s." % config_file
+        sys.exit(-1)
+    _config = config
+
+def process_arguments():
+    parser = optparse.OptionParser(version="%prog 0.1")
+    parser.set_usage("%prog [options]\nRead users from Consul and create them")
+    parser.add_option('-d', '--dry-run', action='store_true', dest='dryrun',
+        help="Show, but do not execute, any commands")
+    (options, args) = parser.parse_args()
+    return options
+
+def readUsersFromConsul():
+    users = {}
+    consul_host = _config.get('consul', 'server')
+    consul_port = _config.get('consul', 'port')
+    consul_scheme = _config.get('consul', 'scheme')
+    consul_path = _config.get('consul', 'path')
+    c = consul.Consul(host=consul_host, port=consul_port, scheme=consul_scheme)
+    x = c.kv.get(consul_path, keys=True, separator='/')
+    if x[1] == None:
```

Comment:
I know python consul returns (index, value) tuples, that's an artifact of Consul, but using
index, consul_user_ids = c.kv.get('foo') instead would be much more reabable than x[1]